sdk/typescript/runtime/module.go

```diff
@@ -149,7 +149,7 @@ func (m *moduleRuntimeContainer) configurePackageJSON(file *dagger.File) *dagger
 	if m.cfg.packageJSONConfig != nil {
 		_, ok := m.cfg.packageJSONConfig.Dependencies["typescript"]
 		if !ok {
-			ctr = ctr.WithExec([]string{"npm", "pkg", "set", "dependencies.typescript=^5.5.4"})
+			ctr = ctr.WithExec([]string{"npm", "pkg", "set", "dependencies.typescript=5.9.3"})
```

Comment:
Should be a constant that's not buried here.

Any suggestions for how to make sure we keep this up-to-date? Dependabot won't catch it.

Reply:
Ideally we should keep the same version that dependabot set in the `typescript/sdk` package.json